scripts/west_commands/run_common.py

```diff
@@ -101,6 +101,8 @@ def add_parser_common(command, parser_adder=None, parser=None):
                        help='override default runner from --build-dir')
     group.add_argument('--skip-rebuild', action='store_true',
                        help='do not refresh cmake dependencies first')
+    group.add_argument('--signed', action='store_true',
+                       help='flash the signed application instead')
```

Comment:
Not sure this is the right location.
This makes it appears for both `flash`, `debug`, `debugserver`, `attach` commands.

But only `flash` and `debug` does flashing:

With this PR:
```
$ west help attach
usage: west attach [-h] [-H] [-d DIR] [-r RUNNER] [--skip-rebuild] [--signed]
                   [--board-dir DIR] [--elf-file FILE] [--hex-file FILE]
                   [--bin-file FILE] [--gdb GDB] [--openocd OPENOCD]
                   [--openocd-search DIR]

Like "west debug", but doesn't reflash the program.
...

general options:
...
  --signed              flash the signed application instead

```

Should probably go into the runners override section: https://github.com/zephyrproject-rtos/zephyr/blob/f78c0ab9da6a1421867ada41064e25ba538627fd/scripts/west_commands/run_common.py#L107-L127, or directly into the flash.py https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/west_commands/flash.py.

Runners configuration override  already states not all arguments are respected by all runners:
```
runner configuration overrides:
  Run "west attach --context" for --build-dir specific options.
  Not all runners respect --elf-file / --hex-file / --bin-file, nor use
  gdb or openocd.

```

Reply:
Added the arguments to flash.py and debug.py